src/main/java/org/jenkinsci/plugins/ghprb/GhprbPullRequest.java

```diff
@@ -27,31 +26,35 @@
 public class GhprbPullRequest {
 
     private static final Logger logger = Logger.getLogger(GhprbPullRequest.class.getName());
+    
+    @Deprecated @SuppressWarnings("unused") private transient GHUser author;
```

Comment:
If the fields are removed then Jenkins will toss all of the old entries and build every PR again.